app/javascript/react/screens/App/Overview/Overview.js

```diff
@@ -412,6 +461,16 @@ Overview.propTypes = {
   yesToDeleteInfrastructureMapping: PropTypes.bool,
   fetchProvidersAction: PropTypes.func,
   isFetchingProviders: PropTypes.bool,
-  hasSufficientProviders: PropTypes.bool
+  hasSufficientProviders: PropTypes.bool,
+  confirmModalVisible: PropTypes.bool,
+  confirmModalOptions: PropTypes.object,
+  showConfirmModalAction: PropTypes.func,
+  hideConfirmModalAction: PropTypes.func,
+  fetchArchivedTransformationPlansUrl: PropTypes.string,
+  archivedTransformationPlans: PropTypes.array,
+  allArchivedPlanRequestsWithTasks: PropTypes.array,
+  isFetchingArchivedTransformationPlans: PropTypes.string,
```

Comment:
Was the string intentional here?

I'm seeing...

```
miq_debug.self-1351f771c2cd2fab0d83069fd3f62aadbb7effc11a01ed4942f52308793453be.js?body=1:30 Warning: Failed prop type: Invalid prop `loading` of type `string` supplied to `Spinner`, expected `boolean`.
    in Spinner (created by Overview)
    in Overview (created by Connect(Overview))
```

and given the condition...

```
        <Spinner
          loading={
            !requestsWithTasksPreviouslyFetched &&
            !this.hasMadeInitialPlansFetch &&
            (isFetchingAllRequestsWithTasks ||
              isFetchingProviders ||
              isFetchingTransformationPlans ||
              isFetchingArchivedTransformationPlans ||
              isFetchingTransformationMappings)
          }
```

When `isFetchingArchivedTransformationPlans` is the first truthy value, it gets passed to `loading`.

Not sure if you want an explicit `!! isFetchingArchivedTransformationPlans` to convert to bool, or if this should never have been a string.

Can you take a look please @michaelkro ?

Reply:
Hi @himdel, I've been noticing that error as well but just haven't had the time to fully investigate why that particular value is a string.

In some cases, we have used IDs as flags to isolate loading state, e.g., disable the migrate button for a specific migration plan ListItem after it is clicked.

However, that being said, I will take a look and at the very least put up a PR coercing it to a bool, since that is really all we need in this case.

Thanks for taking the time to report the issue!